lib/player.js

```diff
@@ -1074,6 +1074,11 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
     if (startTime !== undefined) {
       payload.startTime = startTime;
     }
+    const targetLiveLatencyOverride = this.getConfiguration()
+        .streaming.liveCatchUp.targetLiveLatencyOverride;
```

Comment:
should we add the null check?

Reply:
player_configuration.js#125 initialized the value as 0

Reply:
i see, thanks

Reply:
Just found the null check condition on line 1079 is buggy and will fix as well